market-contract/src/main.sw

```diff
@@ -737,12 +721,7 @@ fn cancel_order_internal(order_id: b256) {
 #[storage(read, write)]
 fn increase_user_volume(user: Identity, volume: u64) {
     extend_epoch();
-    storage
-        .user_volumes
-        .get(user)
-        .try_read()
-        .unwrap_or(UserVolume::new())
-        .update(storage.epoch.read(), volume);
+    let _ = storage.user_volumes.get(user).try_read().unwrap_or(UserVolume::new()).update(storage.epoch.read(), volume);
```

Comment:
Silences compiler warning, because we don't implicitly return the value from `update()`